### DISCUSSION TITLE
Page reload error

### DISCUSSION THREAD

**Discussion by Augustine-edeh** [community/community#71899] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Good day, all.

I have a Reaxt.js project hosted on GitHub Pages. And I'm having this issue where after reloading a page on the browser it displays this GitHub error page (as in the image attached below). I've tried troubleshooting this problem on my own but to no avail. Thought I might get the right help in the community, Thanks for the help in advance🙏

![Screenshot from 2023-10-23 14-08-18](https://github.com/community/community/assets/58579127/8a68942d-a33a-43dc-a7ac-002003df9bfc)

**Comment by queenofcorgis** [comment]
Hi there 👋🏼, 

The community may be better to assist if you include the repo, links, or screenshots ❤️
